python/catboostserver/Makefile

```
@@ -0,0 +1,12 @@
+
+dev_install:
+	uv sync --group test
+
+install_dependencies:
+	uv sync --group test
+
+test: type_check
+	pytest -W ignore
+
+type_check:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)